### PR TITLE
Move elastic index

### PR DESCRIPTION
**Pull request by yildizfatih**
N/A

### CODE REVIEW

**Review by taylanisikdemir — APPROVED** [review]

File: SUMMARY.md [hunk 1]
```diff
@@ -22,4 +22,5 @@
   * [Linux Configuration](examples/linux-configuration.md)
   * [Windows Configuration](examples/windows-configuration.md)
   * [Docker Configuration](examples/docker-configuration.md)
-
+* [Appendix](appendix/README.md)
```
Comment (taylanisikdemir) [inline SUMMARY.md]: appendices/README.md

File: SUMMARY.md [hunk 2]
```diff
@@ -22,4 +22,5 @@
...
+  * [Elastic Index Setup](appendix/elastic-index.md)
```
Comment (taylanisikdemir) [inline SUMMARY.md]: appendices/elastic-index.md

Comment (yildizfatih) [reply SUMMARY.md]: fixed.

File: SUMMARY.md (hunk 1 above)
Comment (yildizfatih) [reply SUMMARY.md]: fixed.

**Pull request merged**